plugins/wallabag.koplugin/main.lua

```diff
@@ -132,6 +136,13 @@ function Wallabag:addToMainMenu(menu_items)
                             self:editServerSettings()
                         end,
                     },
+                    {
+                        text = _("Configure Wallabag client"),
+                        keep_menu_open = true,
+                        callback = function()
```

Comment:
Just self.editClientSettings doesn't work here?

Reply:
Have to admit I haven't tried, just to be consistent with the `editServerSettings` call a few lines above.

But after a quick edit, it doesn't seem to work (the function is called, but the settings dialog is empty). I'm quite new to Lua, but I suppose the `self` object has been lost in the callback?

Reply:
Or rather, `self` would be referring to something else by that time. ;-)